source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -129,6 +129,9 @@ let parse_models_and_queries_from_sources
     ~source_sink_filter
     ~callables
     ~stubs
+    ~python_major_version
+    ~python_minor_version
+    ~python_micro_version
```

Comment:
To improve the readability, could we create a module for this in `modelParser.ml` and `modelParser.mli` and move your current parsing logic here? Maybe something like

```
module PythonVersion = struct
  type t = {
    major_version: int;
    minor_version: int;
    micro_version: int;
  }
  
  (* Move your current parsing logic in modelParser.ml from line 3273 to line 3394 here *)
end
```